docker/templates/build/Dockerfile

```diff
@@ -21,30 +21,20 @@ RUN apt-get update && \
 
 # Install JRuby (the Ubuntu JRuby package is 1.5.6!)
 RUN mkdir -p /var/lib/jruby \
-    && curl -SL http://jruby.org.s3.amazonaws.com/downloads/1.7.20/jruby-bin-1.7.20.tar.gz \
+    && curl -SL http://jruby.org.s3.amazonaws.com/downloads/1.7.26/jruby-bin-1.7.26.tar.gz \
     | tar -z -x --strip-components=1 -C /var/lib/jruby
 ENV PATH /var/lib/jruby/bin:$PATH
 
-# Install Bundler
-RUN jruby -S gem install bundler
-# Install JBundler
-RUN jruby -S gem install jbundler
-# Install a JavaScript runtime
-RUN jruby -S gem install therubyrhino
-
-# Install Maven (the Ubuntu Maven package is 3.0.5)
-RUN mkdir -p /var/lib/maven \
-    && curl -SL http://mirror.metrocast.net/apache/maven/maven-3/3.3.1/binaries/apache-maven-3.3.1-bin.tar.gz \
-    | tar -z -x --strip-components=1 -C /var/lib/maven
-ENV PATH /var/lib/maven/bin:$PATH
+RUN jruby -S gem install bundler jbundler therubyrhino
 
 # Add killbill user into sudo group
-RUN adduser --disabled-password --gecos '' killbill
-RUN usermod -aG sudo killbill
-RUN echo 'killbill:killbill' | chpasswd
+RUN adduser --disabled-password --gecos '' killbill && \
+    usermod -aG sudo killbill && \
+    echo 'killbill:killbill' | chpasswd && \
+    echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
 
-RUN mkdir -p $KILLBILL_HOME $KILLBILL_CONFIG
-RUN chown -R killbill:killbill $KILLBILL_CONFIG $KILLBILL_HOME /var/lib/jruby
+RUN mkdir -p $KILLBILL_HOME $KILLBILL_CONFIG && \
+    chown -R killbill:killbill $KILLBILL_CONFIG $KILLBILL_HOME
```

Comment:
What about `/var/lib/jruby`? Could it be the source of your permission issues when installing gems?